app/src/main/res/layout/activity_login.xml

```diff
@@ -18,82 +20,87 @@
-                    android:layout_height="@dimen/layout_padding_75dp"
-                    android:layout_width="wrap_content"
-                    android:src="@drawable/mifos_logo_new"/>
-
-                <TextView
-                    android:gravity="center"
-                    android:layout_height="wrap_content"
-                    android:layout_width="match_parent"
-                    android:text="@string/enter_you_credentials"/>
+                android:layout_height="wrap_content"
+                android:text="@string/mifos_mobile_cn"
+                android:textAlignment="center"
+                android:textColor="@color/white"
+                android:textSize="35sp"/>
 
-            </LinearLayout>
 
             <LinearLayout
                 android:layout_height="wrap_content"
-                android:layout_marginTop="@dimen/layout_padding_30dp"
+                android:layout_marginTop="@dimen/layout_padding_65dp"
                 android:layout_width="match_parent"
                 android:orientation="vertical">
 
 
                 <com.google.android.material.textfield.TextInputLayout
+                    style="@style/TextInputLayoutLogin"
                     android:layout_height="wrap_content"
-                    android:layout_width="match_parent">
+                    android:layout_width="match_parent"
+                    app:boxStrokeColor="@color/white"
+                    android:textColorHint="@color/white"
+                    android:layout_marginBottom="@dimen/layout_padding_20dp"
+                    android:backgroundTint="@color/white">
 
-                    <EditText
-                        style="@style/Base.TextAppearance.AppCompat.Medium"
-                        android:drawableLeft="@drawable/ic_person_black_24dp"
-                        android:drawablePadding="@dimen/layout_padding_8dp"
-                        android:drawableStart="@drawable/ic_person_black_24dp"
+                    <com.google.android.material.textfield.TextInputEditText
                         android:hint="@string/username"
                         android:id="@+id/etUsername"
                         android:inputType="text"
                         android:layout_height="wrap_content"
-                        android:layout_width="match_parent"/>
+                        android:layout_width="match_parent"
+                        android:textColor="@color/white" />
 
                 </com.google.android.material.textfield.TextInputLayout>
 
                 <com.google.android.material.textfield.TextInputLayout
+                    style="@style/TextInputLayoutLogin"
                     android:layout_height="wrap_content"
                     android:layout_width="match_parent"
-                    app:passwordToggleEnabled="true">
+                    app:passwordToggleEnabled="true"
+                    app:passwordToggleTint="@color/white"
+                    app:boxStrokeColor="@color/white"
```

Comment:
do we need this line of code if you have already specified `boxStrokeColor` in `styles.xml`?